ts/utils/brightness.ts

```diff
@@ -0,0 +1,16 @@
+import { Platform } from "react-native";
+import ScreenBrightness from "react-native-screen-brightness";
+
+export const getBrightness = (): Promise<number> => {
```

Comment:
We should use [TaskEither](https://dev.to/ksaaskil/using-fp-ts-for-http-requests-and-validation-131c) instead.